fairscale/__init__.py

```diff
@@ -4,7 +4,7 @@
 # LICENSE file in the root directory of this source tree.
 
 # Please update the doc version in docs/source/conf.py as well.
-__version__ = "0.1.7"
+__version__ = "0.1.8"
```

Comment:
can you please update docs/source/conf.py as well?

Reply:
perhaps version 0.3 to signify all 3 zeros are there?

Reply:
ahah, as you prefer, I think it goes beyond zero or else, these concepts are not completely new anyway. Doing that for the tongue in cheek reference